doc/platforms/index.rst

```diff
@@ -1,15 +1,17 @@
 .. _platforms-index:
 
-Platform-Specific Notes
-=======================
-
-Contents:
-
-.. toctree::
-   :hidden:
+Platforms-Specific Notes
```

Comment:
The 's' in Platforms doesn't parse for me.  Is it meant to be possessive?  If plural, I'd think that "Notes" would be all that would need to be pluralized, not "Platform-specific" (which I think serves as an adjective -- "what kind of notes"?)